packages/manager-react-components/src/components/datagrid/datagrid.component.tsx

```diff
@@ -90,7 +90,7 @@ export const Datagrid = <T,>({
   className,
   onPaginationChange,
   onSortChange,
-  contentAlignLeft,
+  contentAlignLeft = true,
```

Comment:
let's replace with contentCentered = false instead ?
If we want content aligned to the left by default

Or maybe contentAlign that takes 'left' by default
```
contentAlign: 'left' | 'right' | 'center',
```